client/public/static/js/recipes/recipe_details.js

```diff
@@ -423,14 +400,21 @@ function RecipeDetailsView() {
       urlAction = PUT_ACTION;
       request = {
         recipeName: recipeName,
-        recipeIngredients: recipeIngredients.join(", "),
-        recipeDirections: recipeDirections.join(". "),
```

Comment:
@CamronOlsen see the changes here. Did not see anything breaking when displaying the data on recipe details or when export. Essentially just adding each line or whatever to the array an saving it